vdirsyncer/storage/google.py

```diff
@@ -94,6 +94,11 @@ def _save_token(token):
 class GoogleCalendarStorage(dav.CaldavStorage):
     __doc__ = '''Google calendar.
 
+    .. note:: You have to poperly set up ``collections``, otherwise errors like
```

Comment:
If that's feasible, yeah.
Should I drop the changes to `google.py` from that PR then?